archivist/notebooks/Find Asset and Event Creation.ipynb

```diff
@@ -0,0 +1,329 @@
+    "#\n",
+    "# RKVST Python SDK: https://github.com/rkvst/rkvst-python\n",
+    "#"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "40624149",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from json import dumps as json_dumps\n",
+    "from os import getenv\n",
+    "from warnings import filterwarnings\n",
+    "\n",
+    "from archivist.archivist import Archivist\n",
+    "from archivist.proof_mechanism import ProofMechanism\n",
+    "from archivist.logger import set_logger\n",
+    "\n",
+    "filterwarnings(\"ignore\", message=\"Unverified HTTPS request\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "1f799f2a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# URL, CLIENT, SECRET are environment variables that represent connection parameters.\n",
+    "#\n",
+    "# URL = represents the url to the RKVST application\n",
+    "# CLIENT = represents the client ID from an Application Registration\n",
+    "# SECRET = represents the client secret from an Application Registration\n",
+    "RKVST_URL = getenv(\"URL\")\n",
+    "APPREG_CLIENT = getenv(\"CLIENT\")\n",
+    "APPREG_SECRET = getenv(\"SECRET\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "aa63d95a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_artist(arch, name, artistid):\n",
+    "    \"\"\"\n",
+    "    Finds existing Artist asset by name and unique id\n",
+    "    \"\"\"\n",
+    "    attrs = {\n",
+    "        \"arc_display_type\": \"Artists\",\n",
+    "        \"arc_display_name\": name,\n",
+    "        \"artistid\": artistid,\n",
+    "    }\n",
+    "\n",
+    "    return arch.assets.read_by_signature(attrs=attrs)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "66df6ebc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def create_release(arch, asset, album_name, release):\n",
+    "    \"\"\"\n",
+    "    Creates an event for existing Artist asset\n",
+    "    \"\"\"\n",
+    "    props = {\"operation\": \"Record\", \"behaviour\": \"RecordEvidence\"}\n",
+    "    attrs = {\n",
+    "        \"arc_description\": \"Artist Information for existing Artist\",\n",
+    "        \"arc_display_type\": \"Album Release for existing Artist\",\n",
+    "        \"album_name\": album_name,\n",
+    "        \"relase_year\": release,\n",
+    "    }\n",
+    "\n",
+    "    return arch.events.create(asset[\"identity\"], props=props, attrs=attrs)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "6f5b4221",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Connecting to RKVST\n",
+      "URL https://app.rkvst.io\n"
+     ]
+    }
+   ],
+   "source": [
+    "\"\"\"\n",
+    "Main function of asset and event creation.\n",
```

Comment:
Consider a clearer description: this one isn't creating an Asset

Reply:
Main function descriptions updated for all notebooks